File: python/easy/remove_element.py
def removeElement(nums, val):
  i = 0

  while i < len(nums):
    if nums[i] == val:
      nums.remove(nums[i])
    else:
    i += 1



nums = [0,1,2,2,3,0,4,2]
val = 2
removeElement(nums, val)

File: javascript/easy/remove_element.js
// 27. Remove Element
// https://leetcode.com/problems/remove-element/

var removeElement = function(nums, val) {
    var i = 0;
    
    while (i < nums.length) {
        if (nums[i] == val) {
            nums.splice(i, 1);
        } else {
            i++;
        }
    }
};